src/Lighthouse.Tests.Benchmark/Lighthouse.Tests.Benchmark.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Tmds.ExecFunction" Version="0.5.0" />
+      <PackageReference Include="Universe.CpuUsage" Version="2.2.497.598" />

```

Comment:
Nice, this was the package we mentioned in https://github.com/akkadotnet/akka.net/issues/5400#issuecomment-981870230